tests/landIce/FO_ISMIP/input_fo_ismip-hom_testB.yaml

```diff
@@ -142,15 +142,15 @@ ANONYMOUS:
   Regression For Response 0:
     Relative Tolerance: 1.0e-06
     Absolute Tolerance: 1.0e-06
-    Test Value: 1.067962127873e+01
+    Test Value: 1.618727420764e+01
   Regression For Response 1:
     Relative Tolerance: 1.0e-06
     Absolute Tolerance: 1.0e-10
-    Test Value: 0.0
+    Test Value: 2.372857585175e+00
```

Comment:
this seems like a pretty big change but your code seems more general and I'm not really following what the original intention of the code was for the ismip tests. maybe @ikalash or @mperego could explain.

Reply:
@jewatkins does my argument about the origin of the big regression test diff seem reasonable? If so, I can merge this PR. Otherwise, we need to dig more.